UI/web-src/ngMango/components/userTable/userTable.html

```diff
@@ -50,24 +56,30 @@
                                 <md-menu-divider ng-if-end></md-menu-divider>
 
                                 <md-menu-item>
-                                    <md-button ng-click="$ctrl.showFilters = !$ctrl.showFilters; $ctrl.showFiltersChanged()">
-                                        <md-checkbox ng-click="$event.stopPropagation()" ng-model="$ctrl.showFilters" ng-change="$ctrl.showFiltersChanged()">
+                                    <md-button
+                                        ng-click="$ctrl.showFilters = !$ctrl.showFilters; $ctrl.showFiltersChanged()">
+                                        <md-checkbox ng-click="$event.stopPropagation()" ng-model="$ctrl.showFilters"
+                                            ng-change="$ctrl.showFiltersChanged()">
                                             <span ma-tr="ui.app.showFilters"></span>
                                         </md-checkbox>
                                     </md-button>
                                 </md-menu-item>
-                                
+
                                 <md-menu-item>
                                     <md-menu>
-                                        <md-button ng-click="$mdMenu.open($event)"><span ma-tr="ui.app.selectColumns"></span></md-button>
-                                        <md-menu-content width="3" class="ma-user-table ma-checkbox-menu" md-colors="::{background: 'background-hue-2'}">
+                                        <md-button ng-click="$mdMenu.open($event)"><span
+                                                ma-tr="ui.app.selectColumns"></span></md-button>
+                                        <md-menu-content width="3" class="ma-user-table ma-checkbox-menu"
+                                            md-colors="::{background: 'background-hue-2'}">
                                             <md-menu-item ng-repeat="column in $ctrl.columns track by column.name">
                                                 <md-button ng-init="model = $ctrl.getSelectedColumnsModel(column)"
-                                                        ng-click="model.value = !model.value; $ctrl.selectedColumnsChanged()" md-prevent-menu-close
+                                                    ng-click="model.value = !model.value; $ctrl.selectedColumnsChanged()"
+                                                    md-prevent-menu-close
+                                                    ng-disabled="model.value && $ctrl.selectedColumns.length === 1">
+                                                    <md-checkbox ng-click="$event.stopPropagation()"
+                                                        md-prevent-menu-close md-no-ink="true" ng-model="model.value"
+                                                        ng-change="$ctrl.selectedColumnsChanged()"
```

Comment:
Please avoid reformatting lines that have not been modified. This makes it really hard for me to look at and tell what you have actually changed.